### PR TITLE
PIC-435 add snyk builds and upgrade libraries for high severity CVEs

### PR DESCRIPTION
**Pull request by tomsmyers**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -138,38 +155,49 @@ jobs:
...
+          condition: << parameters.master >>
+          steps:
+            - run:
+                name: Publish to docker hub
+                command: |
+                  docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+                  docker tag mojdigitalstudio/court-case-service:$APP_VERSION mojdigitalstudio/court-case-service:latest
+                  docker push mojdigitalstudio/court-case-service:$APP_VERSION
+                  docker push mojdigitalstudio/court-case-service:latest
+                  fi
```
Comment (chris-faulkner-mojdt) [inline .circleci/config.yml:174]: Looks strange to have this straggling "fi" closing a branch. IS that needed now you have the condition: above?

File: .circleci/config.yml [hunk 2]
```diff
@@ -138,38 +155,49 @@ jobs:
...
       - snyk/scan:
-          fail-on-issues: false
-          monitor-on-build: true
           project: '${CIRCLE_PROJECT_REPONAME}/${CIRCLE_BRANCH}'
-          organization: 'digital-probation-services'
+          monitor-on-build: << parameters.master >>
+          <<: *snyk_options
 
-  image-scan:
+  image_scan:
```
Comment (chris-faulkner-mojdt) [inline .circleci/config.yml:189]: why underscore - next job uses hyphen ?

**Comment by chris-faulkner-mojdt** [comment]
A couple of comments. Looks really good overall and good to have it in

File: .circleci/config.yml (hunk 1 above)
Comment (tomsmyers) [reply .circleci/config.yml:174]: good spot, this is a mistake

File: .circleci/config.yml (hunk 2 above)
Comment (tomsmyers) [reply .circleci/config.yml:189]: i was trying to improve consistency with job naming e.g. `build_docker` `deploy_whatever`. but i can see there are jobs with dashes too `request-preprod-approval`, `sonar-scan`.

Comment (chris-faulkner-mojdt) [reply .circleci/config.yml:189]: I agree - we need to get some consistency

Comment (tomsmyers) [reply .circleci/config.yml:188]: changed the other jobs to use underscores too

File: .circleci/config.yml (hunk 1 above)
Comment (tomsmyers) [reply .circleci/config.yml]: fixed

**Pull request merged**